src/events/message-reaction-add/handlers/request-training.ts

```diff
@@ -65,7 +64,7 @@ export class RequestTrainingHandler implements EventHandler {
                     return;
                 }
 
-                const clickableUrl = `${this.calendarUrl}${date.format('YYYYMMDD')}T100000Z%2F${date.format('YYYYMMDD')}T220000Z`; 
+                const clickableUrl = `${this.calendarUrl}${date.format('YYYYMMDD')}T100000Z%2F${date.format('YYYYMMDD')}T220000Z`;
```

Comment:
maybe we could name this something more descriptive... such as googleCalLink or clickableGoogleCalLink?

Reply:
When I named this, I explicitly tried to avoid being specific to a calendar, like google ... If the user sets the calendarUrl to a nonGoogle url, this would no longer be a google url.

That said, the generated href is probably specific to Google, but it could work with other calendars too. We should introduce calendar enums (Google, Outlook, etc) and then based on the enum, generate a link ... but I didn't see importance in this yet

Reply:
I see what you mean, I suppose `clickableUrl` followed by an assignment of `calendarUrl` is enough for someone to understand what's going on.
I like that idea, but as you said, probably better for later down the road.